### ISSUE TITLE
Feature Request: convert the resultant xcresult to a list of passed/failed tests

### ISSUE THREAD

**Issue by lyndsey-ferguson** [lyndsey-ferguson/fastlane-plugin-test_center#218]
#### Motivation Behind Feature

People don't always want junit results, and they want to use the xcresult bundles. See this tweet:

<img width="588" alt="Screen Shot 2020-04-28 at 4 29 25 PM" src="https://user-images.githubusercontent.com/5741516/80534575-75433200-896d-11ea-9d8e-57af529dcc52.png">

#### Feature Description

Add a new action that can return a list of failed/passed tests.

**Comment by lyndsey-ferguson** [comment]
[Here](https://www.chargepoint.com/engineering/xcparse/) is an article describing the steps that have to be performed to get the actual data. Starting with:
```
xcrun xcresulttool get --path path/to/xcresult --format json
```

**Comment by lyndsey-ferguson** [comment]
I've created the project [Get tests from xcresult](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/projects/5)

**Comment by lyndsey-ferguson** [comment]
I created the new action to get test results from your xcresult bundles! 🎉 

Check it out [here](https://t.co/d4bVMWuxSo?amp=1)
![Screen Shot 2020-07-30 at 7 55 48 PM](https://user-images.githubusercontent.com/5741516/88985689-f547a380-d29e-11ea-9b48-ecb8f9a05881.png)

**Comment by lyndsey-ferguson** [comment]
@vdavydovHH this one is for you, you have early access!

**Comment by lyndsey-ferguson** [comment]
Released in [v3.14.2](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.14.2)
